tools/AzPreview/AzPreview.psd1

```diff
@@ -52,43 +52,44 @@ DotNetFrameworkVersion = '4.7.2'
 # ProcessorArchitecture = ''
 
 # Modules that must be imported into the global environment prior to importing this module
-RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '2.7.4'; }, 
+RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '2.7.5'; }, 
             @{ModuleName = 'Az.ADDomainServices'; RequiredVersion = '0.1.0'; }, 
             @{ModuleName = 'Az.Advisor'; RequiredVersion = '1.1.2'; }, 
             @{ModuleName = 'Az.Aks'; RequiredVersion = '3.1.2'; }, 
             @{ModuleName = 'Az.AlertsManagement'; RequiredVersion = '0.4.0'; }, 
             @{ModuleName = 'Az.AnalysisServices'; RequiredVersion = '1.1.4'; }, 
             @{ModuleName = 'Az.ApiManagement'; RequiredVersion = '2.3.1'; }, 
-            @{ModuleName = 'Az.AppConfiguration'; RequiredVersion = '1.0.0'; }, 
-            @{ModuleName = 'Az.ApplicationInsights'; RequiredVersion = '1.3.0'; }, 
+            @{ModuleName = 'Az.AppConfiguration'; RequiredVersion = '1.1.0'; }, 
+            @{ModuleName = 'Az.ApplicationInsights'; RequiredVersion = '1.3.1'; }, 
             @{ModuleName = 'Az.Attestation'; RequiredVersion = '1.0.0'; }, 
             @{ModuleName = 'Az.Automation'; RequiredVersion = '1.7.3'; }, 
             @{ModuleName = 'Az.BareMetal'; RequiredVersion = '0.1.0'; }, 
             @{ModuleName = 'Az.Batch'; RequiredVersion = '3.1.1'; }, 
             @{ModuleName = 'Az.Billing'; RequiredVersion = '2.0.0'; }, 
             @{ModuleName = 'Az.Blueprint'; RequiredVersion = '0.4.0'; }, 
             @{ModuleName = 'Az.BotService'; RequiredVersion = '0.3.2'; }, 
-            @{ModuleName = 'Az.Cdn'; RequiredVersion = '1.8.1'; }, 
+            @{ModuleName = 'Az.Cdn'; RequiredVersion = '1.8.2'; }, 
             @{ModuleName = 'Az.ChangeAnalysis'; RequiredVersion = '0.1.0'; }, 
             @{ModuleName = 'Az.CloudService'; RequiredVersion = '1.1.0'; }, 
             @{ModuleName = 'Az.CognitiveServices'; RequiredVersion = '1.10.0'; }, 
             @{ModuleName = 'Az.Communication'; RequiredVersion = '0.2.0'; }, 
-            @{ModuleName = 'Az.Compute'; RequiredVersion = '4.24.1'; }, 
+            @{ModuleName = 'Az.Compute'; RequiredVersion = '4.25.0'; }, 
+            @{ModuleName = 'Az.ConfidentialLedger'; RequiredVersion = '0.1.0'; }, 
```

Comment:
Please update our module list file to include this module after release

Reply:
Sure, will do